app/webserver/helpers.go

```diff
@@ -240,36 +240,42 @@ func joinGroup(joinGroupmessage JoinGroupMessage) *store.Session {
 	log.Infoln("[axolotl] joinGroup", joinGroupmessage.ID)
 	group, err := textsecure.JoinGroup(joinGroupmessage.ID)
 	if err != nil {
+		log.Errorln("[axolotl] joinGroup failed", err)
 		// update group also in the case that the group is already joined to remove the join message
```

Comment:
You may want to also drop this comment ;)

Reply:
Is this better?